xbmc/addons/kodi-addon-dev-kit/include/kodi/addon-instance/Peripheral.h

```diff
@@ -674,8 +673,8 @@ namespace addon
     /*!
      * @brief Notify the frontend that button maps have changed
      *
-     * @param[optional] deviceName The name of the device to refresh, or empty/null for all devices
-     * @param[optional] controllerId The controller ID to refresh, or empty/null for all controllers
+     * @param[optional] strDeviceName The name of the device to refresh, or empty/null for all devices
+     * @param[optional] strControllerId The controller ID to refresh, or empty/null for all controllers
      */
     void RefreshButtonMaps(const std::string& strDeviceName = "", const std::string& strControllerId = "")
```

Comment:
Can you actually go in the other direction, and remove the `str` prefixes here?

Reply:
Sure, it's nicer without the prefix.